pmd-core/src/main/java/net/sourceforge/pmd/cli/PMDCommandLineInterface.java

```diff
@@ -60,10 +60,10 @@ public static String buildUsageText() {
     public static String buildUsageText(JCommander jcommander) {
         StringBuilder usage = new StringBuilder();
 
-        String allCommandsDescription = null;
+        StringBuilder allCommandsDescription = null;
         if (jcommander != null && jcommander.getCommands() != null) {
             for (String command : jcommander.getCommands().keySet()) {
-                allCommandsDescription += jcommander.getCommandDescription(command) + PMD.EOL;
+                allCommandsDescription.append(jcommander.getCommandDescription(command)).append(PMD.EOL);
```

Comment:
@XenoAmess This introduces a NPE.

Reply:
@tweimer 
my bad.
btw: is this funtion covered by any test?

Reply:
> @XenoAmess This introduces a NPE.

@tweimer fixed in https://github.com/pmd/pmd/pull/2745.